assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -130,6 +132,42 @@ const baseResolvers = {
 	},
 };
 
+const baseSelectors = {
+	/**
+	 * Checks whether the provided audiences are available.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object}               state                 Data store's state.
+	 * @param {string|Array<string>} audienceResourceNames Custom dimensions to check.
+	 * @return {boolean} True if all provided audiences are available, otherwise false. Undefined if available audiences are not loaded yet.
+	 */
+	haveAudiences: createRegistrySelector(
```

Comment:
Does `hasAudiences` sound more grammatically correct, as audiences belong to a singular property entity?